libs/ui/src/lib/list/table/sorts.ts

```diff
@@ -18,4 +18,9 @@ export const sorts = {
     if (a.length < b.length) return -1;
     return 0;
   },
+  byTitle: (a: Movie, b: Movie) => {
+    if (!a.title.international) return -1;
+    if (!b.title.international) return 1;
```

Comment:
To get sort to work with 2 movies, I had to update like this:
```suggestion
    if (!a?.title.international) return -1;
    if (!b?.title.international) return 1;
```